index.bs

```diff
@@ -15,6 +15,8 @@ spec:infra; type:dfn; for:/; text:list
 spec:infra; type:dfn; for:list; text:append
 spec:infra; type:dfn; for:set; text:append
 spec:html; type:dfn; for:/; text:origin
+spec:html; type:dfn; for:environment settings object; text:global object
+spec:html; type:dfn; for:/; text:browsing context
```

Comment:
I hesitated between "for:/" and "for:Window". If we go with the latter, do we also need to verify that the global object is a Window?

Reply:
To be clear, I think it's a safe assumption. Just wondering if we should make it explicit and assert it.

Reply:
This would need to be for: Window. You can't link to the generic concept of "X" when you're doing "global object's X".

In general this feels like a big abuse of link-defaults though. What if sometimes you want to link to the generic concept and sometimes to the per-Window field? I'd suggest deleting these lines and being explicit at the usage site, via `[=Window/browsing context=]`. Same for `[=environment settings object/global object=]`.

Reply:
Cool, done